content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/v2/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What makes it necessary to introduce a new youtube embeddable (v2)? Couldn't we just update the current youtube embeddable code at `.../embed/embeddable/youtube`?